android/build.gradle

```diff
@@ -20,5 +20,5 @@ android {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
```

Comment:
What about version of React Native? Should we lock this to currently last supported too?
Imho, i don't think so, but am just raising question.